### ISSUE TITLE
[MHR:S] High RAM and VRAM usage?

### ISSUE THREAD

**Issue by JPoHondaBing** [praydog/REFramework#541]
Hi, ever since I installed the update to REFramework for TU2, I've been experiencing some drastic increase in RAM and VRAM usage to the point where my computer freezes. As suggested by another user with a similar issue here, I changed the option under "Garbage Collection Stats" to "Managed by Lua." That seemed to fix the high RAM usage/memory leak issue. However, I'm still experiencing high VRAM usage.

[re2_framework_log.txt](https://github.com/praydog/REFramework/files/9736324/re2_framework_log.txt)

Edit: Switching from 1.35 back to 1.34 seems to fix the problem

**Comment by morerokk** [comment]
Same issue here, it seems to be related to shader compiling possibly?

**Comment by JPoHondaBing** [comment]
not sure about that, since Reshade stopped working for me since TU2 so I stopped using it. I still had the problem with 1.35 after I uninstalled Reshade though.

I suppose I should add this here: after switching to 1.34 it felt fine for a bit, but last week I noticed VRAM was high again after a few hours of gameplay. Decided to tinker around and found out that if I have the REFramework menu bar up my VRAM decreases by about 50% and stays around there compared to when I hide it by pressing INSERT on keyboard
